### PR TITLE
PR #22 (modify/loadConfigFromFS)

### PR DESCRIPTION
**Pull request by rgadon107**
@hellofromtonya Following our conversation earlier today (07/02/19), I wrote a new unit test for `_the_store()` to test that a new config assigned to the same store key will overwrite the initial (existing) config. The edge cases for the unit & integration test of `loadConfigFromFilesystem` that will never occur were both removed.

### CODE REVIEW

File: mu-plugins/central-hub/tests/phpunit/unit/config-store/_theStore.php [hunk 1]
```diff
@@ -107,4 +107,25 @@ public function test_should_return_stored_config_when_given_valid_store_key() {
...
+		$this->assertTrue( _the_store( 'foo', $config ) );
+
+		$new_config = [
+			'aaa' => 37,
+			'ccc' => 'Coding is fun!',
+			'eee' => 'WordPress rocks!',
+		];
+
+		$this->assertTrue( _the_store( 'foo', $new_config ) );
+	}
```
Comment (hellofromtonya) [inline mu-plugins/central-hub/tests/phpunit/unit/config-store/_theStore.php]: @rgadon107 The test validates that the config was stored.  But it doesn’t validate the config was overwritten with the new config. You need to go one more step and get the config out of the store.  Make sure it actually overwrote it.

Comment (rgadon107) [reply mu-plugins/central-hub/tests/phpunit/unit/config-store/_theStore.php]: @hellofromtonya 
>But it [the test] doesn’t validate the config was overwritten with the new config. 

You're right. When I went to bed last night, I knew I hadn't conclusively demonstrated that the config was overwritten. See next commit below (344d351), which addresses your comment.
